crates/bifrost/src/bifrost.rs

```diff
@@ -92,6 +93,13 @@ impl Bifrost {
     ///
     /// It's recommended to use the [`Appender`] interface. Use [`Self::create_appender`]
     /// and reuse this appender to create a sequential write stream to the virtual log.
+    #[instrument(
+        level="trace",
+        skip(self, body),
+        fields(
+            otel.name = "bifrost: append",
```

Comment:
`name="bifrost: append"` is not enough to let it appear in OTEL?

Reply:
Maybe `Bifrost::append` to follow the double colon notation?

Reply:
I am actually wondering if we should keep otel.name separate from `name` since name will also affect logs. While `otel.name` will only reflect in otel collectors like jaeger